org-formation/650-identity-providers/_tasks.yaml

```diff
@@ -831,3 +831,160 @@ GithubOidcOpenChallengesDeploy:
     Account:
       - !Ref ITSandboxAccount
     Region: us-east-1
+
+############################### Managed Policies ###############################
+# Managed policies used in github OIDC providers
+# Note: Managed policies can be used as work around for the AWS cloudformation
+#       Parameter value limit of 4096 characters.
+
+SynapseMonorepoBucketAccessPolicy:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.5.1/templates/IAM/managed-policy.yaml
+  StackName: synmono-bucket-access-policy
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: false
+    Account: !Ref SageITAccount
+    Region: !Ref primaryRegion
+  Parameters:
+    PolicyDocument: |
```

Comment:
since it's json spacing don't matter therefore using `>-` should give you the most compact solution. https://yaml-multiline.info/